yt_dlp/extractor/chzzk.py

```diff
@@ -0,0 +1,134 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    parse_iso8601,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class CHZZKLiveIE(InfoExtractor):
+    IE_NAME = 'chzzk:live'
+    _VALID_URL = r'https?://chzzk\.naver\.com/live/(?P<id>[\da-f]+)'
+    _TESTS = [{
+        'url': 'https://chzzk.naver.com/live/c68b8ef525fb3d2fa146344d84991753',
+        'info_dict': {
+            "id": "c68b8ef525fb3d2fa146344d84991753",
+            "ext": "mp4",
```

Comment:
Use single quotes
```suggestion
            'id': 'c68b8ef525fb3d2fa146344d84991753',
            'ext': 'mp4',
```

Reply:
Sorry, didn't notice the double quotes.
I'm too used to `prettier` that automatically replaces the right quotes on file save.
It's strange that the linter from github actions doesn't argue about this.

Reply:
> It's strange that the linter from github actions doesn't argue about this.

Yeah, this is something we are looking to improve.